src/r_draw.h

```diff
@@ -48,7 +48,15 @@ void R_SetFuzzPosTic(void);
 void R_SetFuzzPosDraw(void);
 
 // [FG] spectre drawing mode
-extern boolean fuzzcolumn_mode;
+typedef enum
+{
+    FUZZ_ORIGINAL,
```

Comment:
Why? It doesn't cost anything and other ports with increased resolution still look like this.

Reply:
If there is any performance benefit, it can be used internally at 1x scale still, but I don't see any reason to expose it as a menu option these days.

Reply:
Removed in a14a89b